cloudlift/config/environment_configuration.py

```diff
@@ -119,9 +119,14 @@ def _create_config(self):
         cluster_min_instances = prompt("Min instances in cluster", default=1)
         cluster_max_instances = prompt("Max instances in cluster", default=5)
         cluster_instance_type = prompt("Instance type", default='m5.xlarge')
+        cluster_ami_id_ssm = prompt("SSM parameter for Custom AMI ID for cluster", default='None')
```

Comment:
Prompt msg need improvement. we should ask of Sam parameter path and not AMI ID

Reply:
modified prompt message